services/src/Microsoft.Azure.IIoT.Services.OpcUa.Registry.Sync/src/HostStarterService.cs

```diff
@@ -69,6 +72,7 @@ ILogger logger
             ServiceInfo = serviceInfo ?? throw new ArgumentNullException(nameof(serviceInfo));
             _hostProcesses = hostProcesses?.ToList() ?? throw new ArgumentNullException(nameof(hostProcesses));
             _logger = logger ?? throw new ArgumentNullException(nameof(logger));
+            _metricServer = new KestrelMetricServer(port: kSyncPrometheusPort);
```

Comment:
Please wrap creation, start and stop of `KestrelMetricServer` in an `IHostProcess` interface similar to other processes that we have. That way you would only need to register it with `ContainerBuilder` in `Program.cs` and the same code would be used for all processors. Port can be passes as a configuration.